imod/mf6/rch.py

```diff
@@ -152,3 +164,69 @@ def _validate(self, schemata, **kwargs):
 
     def get_regrid_methods(self) -> Optional[dict[str, Tuple[RegridderType, str]]]:
         return self._regrid_method
+
+    @classmethod
+    def from_imod5_data(
+        cls,
+        imod5_data: dict[str, dict[str, GridDataArray]],
+        discretization_package: StructuredDiscretization,
```

Comment:
I think this can be shortened to:

```suggestion
        dis_pkg: StructuredDiscretization,
```

As ``DIS`` is an abbreviation used by MODFLOW6 and ``pkg`` is a very common abbreviation for "package".

Reply:
done